README.md

```diff
@@ -133,7 +133,11 @@ The OPA-Istio plugin supports the following configuration fields:
 | Field | Required | Description |
 | --- | --- | --- |
 | `plugins["envoy_ext_authz_grpc"].addr` | No | Set listening address of Envoy External Authorization gRPC server. This must match the value configured in the Envoy Filter resource. Default: `:9191`. |
-| `plugins["envoy_ext_authz_grpc"].query` | No | Specifies the name of the policy decision to query. The policy decision must be return a `boolean` value. `true` indicates the request should be allowed and `false` indicates the request should be denied. Default: `data.istio.authz.allow`. |
+| `plugins["envoy_ext_authz_grpc"].query` | No | Specifies the name of the policy decision to query. The policy decision can either be a `boolean` or an `object`. If boolean, `true` indicates the request should be allowed and `false` indicates the request should be denied. If the policy decision is an object, it **must** contain the `status` key set to either `true` or `false` to indicate if the request is allowed or not respectively. It can optionally contain a `headers` field to send custom headers to the downstream client or upstream. Also an optional `body` field can be included in the policy decision to send a response body data to the downstream client. Default: `data.istio.authz.allow`.|
+
+If the configuration does not specify the `query` field, `data.istio.authz.allow` will be considered as the default name of the policy decision to query. The policy decision returned by the rule specified by the default `query` is `boolean` in nature.
```

Comment:
I'd remove the last sentence as the semantics do not depend on the name.

Reply:
Removed.